plugin-base/src/main/java/io/everytrade/server/plugin/impl/everytrade/parser/exchange/anycoin/AnycoinBeanV1.java

```diff
@@ -0,0 +1,198 @@
+package io.everytrade.server.plugin.impl.everytrade.parser.exchange.anycoin;
+
+import com.univocity.parsers.annotations.Headers;
+import com.univocity.parsers.annotations.Parsed;
+import io.everytrade.server.model.Currency;
+import io.everytrade.server.model.TransactionType;
+import io.everytrade.server.plugin.api.parser.ImportedTransactionBean;
+import io.everytrade.server.plugin.api.parser.TransactionCluster;
+import io.everytrade.server.plugin.impl.everytrade.parser.exception.DataIgnoredException;
+import io.everytrade.server.plugin.impl.everytrade.parser.exchange.ExchangeBean;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+
+import static io.everytrade.server.plugin.impl.everytrade.parser.ParserUtils.DECIMAL_DIGITS;
+import static io.everytrade.server.plugin.impl.everytrade.parser.exchange.anycoin.AnycoinCurrencySwitcher.SWITCHER;
+import static io.everytrade.server.plugin.impl.everytrade.parser.exchange.anycoin.AnycoinSupportedOperations.UNSUPPORTED_OPERATION_TYPES;
+
+@EqualsAndHashCode(callSuper = true)
+@Data
+@Headers(sequence = {"Date", "Type", "Amount", "Currency", "Order ID"}, extract = true)
+public class AnycoinBeanV1 extends ExchangeBean {
+
+    private Instant date;
+    private String type;
+    private BigDecimal amount;
+    private Currency coin;
+    private String orderId;
+
+    Currency marketBase;
+    BigDecimal baseAmount;
+    Currency marketQuote;
+    BigDecimal quoteAmount;
+    TransactionType transactionType;
+    AnycoinOperationTypeV1 operationType;
+    String currencyEndsWithS;
+
+
+    public String getOrderId() {
+        return orderId;
+    }
+
+    @Parsed(field = "Date")
+    public void setDate(String date) {
+        this.date = Instant.parse(date);
+    }
+
+    @Parsed(field = "Type")
+    public void setType(String type) {
+        if (type == null) {
+           return;
+        }
+        this.operationType = AnycoinOperationTypeV1.getEnum(type.toUpperCase());
+        this.type = type;
+    }
+
+    @Parsed(field = "Amount")
+    public void setAmount(BigDecimal amount) {
+        this.amount = amount;
+    }
+
+    @Parsed(field = "Currency")
+    public void setCoin(String coin) {
+
+        if (coin.endsWith(".S")) {
```

Comment:
Save into one field the origin name of the currency and use it for validation of stakings